CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Changelog
 
+## 1.2.1 (not yet released)
+- added `openstack.projectName`, `openstack.projectDomainId` and `openstack.userDomainName` to the configurable options for OpenStack provider, in order to authenticate when using Keystone v3 API
+- added `openstack.autoAssignFloatingIp` to the configurable options for OpenStack provider
+
 ## 1.2.0 (not yet released)
+- added `exec` method with configurable timeout to `org.wildfly.extras.sunstone.api.process.ExecBuilder`
```

Comment:
this line can be removed as this change has already been released with 1.1.0 :)

Reply:
done!